src/ios/CDVWKWebViewEngine.m

```diff
@@ -139,12 +139,21 @@ - (instancetype)initWithFrame:(CGRect)frame
     return self;
 }
 
-- (void)initWebServer:(NSDictionary*)settings
+- (void)initWebServer
 {
     [GCDWebServer setLogLevel: kGCDWebServerLoggingLevel_Warning];
     self.webServer = [[GCDWebServer alloc] init];
-    [self.webServer addGETHandlerForBasePath:@"/" directoryPath:@"/" indexFilename:nil cacheAge:3600 allowRangeRequests:YES];
```

Comment:
If you take this out here we need to still set some of these options:
cacheAge:3600 allowRangeRequests:YES

Otherwise range requests for locally hosted video or audio will fail and we have no control of the caching behaviour.

Reply:
The line it's in here
https://github.com/ionic-team/cordova-plugin-ionic-webview/blob/master/src/ios/CDVWKWebViewEngine.m#L762
Changed the cache to 0, but allowRangeRequests is still YES